diff --git a/node_modules/css-to-react-native-transform/dist/index.js b/node_modules/css-to-react-native-transform/dist/index.js
index 57d2ceb..ea66ff0 100644
--- a/node_modules/css-to-react-native-transform/dist/index.js
+++ b/node_modules/css-to-react-native-transform/dist/index.js
@@ -38,15 +38,15 @@ var transformDecls = function transformDecls(styles, declarations, result) {
     var declaration = declarations[d];
     if (declaration.type !== "declaration") continue;
     var property = declaration.property;
-    var value = (0, _rem.remToPx)(declaration.value);
+    var value = (0, _rem.remToPx)(declaration.value.replace(' !important', ''));
     var isLengthUnit = lengthRe.test(value);
     var isViewportUnit = viewportUnitRe.test(value);
     var isPercent = percentRe.test(value);
     var isUnsupportedUnit = unsupportedUnitRe.test(value);
 
-    if (property === "line-height" && !isLengthUnit && !isViewportUnit && !isPercent && !isUnsupportedUnit) {
-      throw new Error("Failed to parse declaration \"".concat(property, ": ").concat(value, "\""));
-    }
+    // if (property === "line-height" && !isLengthUnit && !isViewportUnit && !isPercent && !isUnsupportedUnit) {
+    //   throw new Error("Failed to parse declaration \"".concat(property, ": ").concat(value, "\""));
+    // }
 
     if (!result.__viewportUnits && isViewportUnit) {
       result.__viewportUnits = true;
